bundles/com.espressif.idf.launch.serial.ui/src/com/espressif/idf/launch/serial/ui/internal/NewSerialFlashTargetWizardPage.java

```diff
@@ -265,24 +269,27 @@ public TargetPortInfo(String name) {
 		@Override
 		protected IStatus run(IProgressMonitor monitor) {
 			EspToolCommands espToolCommands = new EspToolCommands();
+
+			String message = String.format(Messages.TargetPortUpdatingMessage, port);
 			display.asyncExec(() -> {
-				try {
-					String port = serialPortCombo.getText().split(PORT_NAME_DESCRIPTOR_SPLITOR)[0];
-					String message = String.format(Messages.TargetPortUpdatingMessage, port);
-					if (infoArea != null && !infoArea.isDisposed())
-						infoArea.append(System.lineSeparator() + message);
-					Process chipInfoProcess = espToolCommands.chipInformation(port);
-					InputStream targetIn = chipInfoProcess.getInputStream();
-					BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(targetIn));
-					StringBuilder chipInfo = new StringBuilder();
-					String readLine;
-					while ((readLine = bufferedReader.readLine()) != null) {
+				if (infoArea != null && !infoArea.isDisposed())
+					infoArea.append(System.lineSeparator() + message);
+			});
+			try {
+				Process chipInfoProcess = espToolCommands.chipInformation(port);
+				InputStream targetIn = chipInfoProcess.getInputStream();
+				BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(targetIn));
```

Comment:
spotbugs complain here about reliance on default encoding and that run(IProgressMonitor) may fail to close stream